src/test/java/org/carlspring/cloud/storage/s3fs/S3ClientIT.java

```diff
@@ -0,0 +1,126 @@
+package org.carlspring.cloud.storage.s3fs;
+
+import org.carlspring.cloud.storage.s3fs.util.EnvironmentBuilder;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import software.amazon.awssdk.services.s3.model.PutObjectResponse;
+import static java.util.UUID.randomUUID;
+import static org.carlspring.cloud.storage.s3fs.S3Factory.ACCESS_KEY;
+import static org.carlspring.cloud.storage.s3fs.S3Factory.REGION;
+import static org.carlspring.cloud.storage.s3fs.S3Factory.SECRET_KEY;
+import static org.carlspring.cloud.storage.s3fs.util.EnvironmentBuilder.getRealEnv;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class S3ClientIT
+{
+
+    S3Client client;
+
+
+    @BeforeEach
+    public void setup()
+    {
+        // s3client
+        final Map<String, Object> credentials = getRealEnv();
+
+        final AwsCredentials credentialsS3 = AwsBasicCredentials.create(credentials.get(ACCESS_KEY).toString(),
+                                                                  credentials.get(SECRET_KEY).toString());
```

Comment:
Cosmetic: Indentation here needs to be fixed.

Reply:
Done!